workers/ohsome_quality_analyst/utils/definitions.py

```diff
@@ -211,5 +210,26 @@ def get_report_classes() -> Dict:
     )
 
 
-ReportResult = collections.namedtuple("Result", "label value text")
-ReportMetadata = collections.namedtuple("Metadata", "name description")
+def get_indicator_names() -> List[str]:
+    return list(load_metadata("indicators").keys())
+
+
+def get_report_names() -> List[str]:
+    return list(load_metadata("reports").keys())
+
+
+def get_layer_names() -> List[str]:
+    return list(load_layer_definitions().keys())
+
+
+def get_dataset_names() -> List[str]:
+    return list(DATASETS.keys())
+
+
+def get_fid_fields() -> List[str]:
+    fid_fields = []
+    for _, dataset in DATASETS.items():
+        fid_fields.append(dataset["default"])
+        if "other" in dataset.keys():
+            fid_fields += list(dataset["other"])
+    return fid_fields
```

Comment:
You can be fancy and write this more general, but this is totally not necessary :wink: 
```suggestion
def get_fid_fields() -> List[str]:
    return flatten_sequence(DATASETS)
```

Add this to `helper.py`:
```python
def flatten_sequence(input: Union[dict, list, tuple, set]):
    """Returns the given input sequence as a list.

    If the input is a dict, it returns all values.
    """
    output = []
    if isinstance(input, dict):
        input = input.values()
    for val in input:
        if isinstance(val, (dict, list, tuple, set)):
            output += flatten_sequence(val)
        else:
            output.append(val)
    return output
```

Reply:
I implemented your suggested helper function.